network.go

```diff
@@ -10,31 +10,33 @@ import (
 )
 
 type _Network struct {
-	network                map[string]AccountID
+	network                map[*ManagedNodeAddress]AccountID
 	nodes                  []*_Node
 	networkNodes           map[AccountID]*_Node
 	maxNodeAttempts        int
 	nodeWaitTime           time.Duration
 	maxNodesPerTransaction *int
 	addressBook            map[AccountID]_NodeAddress
 	networkName            *NetworkName
+	transportSecurity      bool
 }
 
 func _NewNetwork() _Network {
 	return _Network{
-		network:                make(map[string]AccountID),
+		network:                make(map[*ManagedNodeAddress]AccountID),
```

Comment:
The original value here was to keep a copy of the network the user submitted. This has been removed in Java SDK and instead a network map type will be constructed on the fly.